cpp/src/structure/create_graph_from_edgelist_impl.hpp

```diff
@@ -108,21 +105,16 @@ create_graph_from_edgelist_impl(raft::handle_t const& handle,
       minor_ptrs[i] = (store_transposed ? edgelist_rows.begin() : edgelist_cols.begin()) +
                       edgelist_displacements[i];
     }
-    std::tie(renumber_map_labels,
-             partition,
-             number_of_vertices,
-             number_of_edges,
-             *vertex_partition_segment_offsets) =
-      cugraph::renumber_edgelist<vertex_t, edge_t, multi_gpu>(
-        handle,
-        local_vertex_span
-          ? std::optional<std::tuple<vertex_t const*, vertex_t>>{std::make_tuple(
-              (*local_vertex_span).data(), static_cast<vertex_t>((*local_vertex_span).size()))}
-          : std::nullopt,
-        major_ptrs,
-        minor_ptrs,
-        edgelist_edge_counts,
-        edgelist_intra_partition_segment_offsets);
+    std::tie(renumber_map_labels, meta) = cugraph::renumber_edgelist<vertex_t, edge_t, multi_gpu>(
```

Comment:
Perhaps time to progressively replace `std::tie()` by C++17 structural bindings, which might be a bit more readable?

Reply:
Yeah... Here I used std::tie instead of structural binding as renumber_map_labels & meta are defined outside the ```{}``` block, but I don't see much need for this block (besides, releasing the host memory hold by ```std::vector<vertex_t const*> major_ptrs & minor_ptrs``` little early... I will update this.